CONTRIBUTING.md

```diff
@@ -71,7 +71,7 @@ git clone https://github.com/selfhostedshow/wiki wiki &&\
 - Start development server on http://localhost:8000:
 
 ```bash
-docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material
+docker-compose up -d
```

Comment:
I know I said to use the -d argument but now I'm having second thoughts. What do you think?

Reply:
Good point. Let's remove `-d` so that way it remains in the foreground and users don't forget to CTRL+C the process.

Reply:
Removed `-d` in 715e7f0f